docs/source/dataanalysis/dataanalysis.rst

```diff
@@ -14,6 +14,40 @@ For data analysis of openPMD data, see examples of `many supported tools, Python
 
 See also `WarpX' documentation on openPMD <https://warpx.readthedocs.io/en/latest/dataanalysis/formats.html>`__.
 
+Additional Beam Attributes
+""""""""""""""""""""""""""
+
+We add the following additional attributes on the openPMD ``beam`` species at the monitor position.
+
+Reference particle:
+
+* ``beta_ref`` reference particle normalized velocity :math:`\beta = v/c`
+* ``gamma_ref`` reference particle Lorentz factor :math:`\gamma = 1/\sqrt{1-\beta^2}`
+* ``s_ref`` integrated orbit path length, in meters
+* ``x_ref`` horizontal position x, in meters
+* ``y_ref`` vertical position y, in meters
+* ``z_ref`` longitudinal position z, in meters
+* ``t_ref`` clock time * c in meters
+* ``px_ref`` momentum in x, normalized to proper velocity :math:`p_x = \gamma v_x`
```

Comment:
Oh, isn't _proper velocity_ (I took this from the ref particle doc string) usually gamma * v?

I would call beta * gamma a normalized _momentum_ (p/mc).

Reply:
I don't know where the doc "proper velocity" originated.  However, beta*gamma is also the proper velocity divided by c.  It would be more clear to say simply:  "momentum in x, normalized by m*c"